source/documentation/support.md

```diff
@@ -1,86 +1,16 @@
-# How to get Support
+# Support
 
-## Summary
+The Analytical Platform team provides support for problems with the platform. This does not include third-party tools and software you can access through the Analytical Platform such as GitHub. You can [see our shared responsibility model](/./shared-responsibility-model.html) for more information about what the Analytical Platform team can help with. 
 
-- The Analytical Platform team in MOJ Digital & Technology is responsible for providing access to software like R Studio, Visual Studio Code and Jupyter.
-- Analysts themselves are responsible for the code they write in these tools, and the Platform team is not responsible for assisting with problems with this code.  As a rule of thumb, if the problem you're experiencing would also occur with R Studio or Python installed on a standalone computer, then the platform team don't offer support.
-- If you've read through the user guidance and are still stuck, the best place to go for support is the [#analytical-platform-support](https://app.slack.com/client/T1PU1AP6D/C4PF7QAJZ) Slack channel for issues with the platform, or the [#r](https://app.slack.com/client/T1PU1AP6D/C1PUCG719) channel and [#python](https://app.slack.com/client/T1PU1AP6D/C1Q09V86S) channel to get support from peers in your code.
+Before you ask for support, make sure you’ve tried the following steps. 
 
-What follows provides further details about how to ask for support, and what you can do to get your issues resolved as quickly as possible.
+## Check for a known fix to your problem 
+First, you should see if your problem is listed in our [common errors and solutions](/./errors.html). If you cannot find it, then search online for your problem or error message if you have one. We also recommend searching for your problem on Slack using relevant keywords. 
```

Comment:
Does `/./errors.html` work? I think you _might_ be able to just send it to `/errors/

Reply:
Yeah, it worked in preview but I can try that too

Reply:
nvm! it does work, just looked strange, hadn't seen any other references using `/./`